sda-commons-starter/src/main/java/org/sdase/commons/starter/SdaPlatformBundle.java

```diff
@@ -0,0 +1,352 @@
+
+    @Override
+    public PlatformBundleBuilder<C> disableBufferLimitValidationSecurityFeature() {
+      this.securityBundleBuilder = SecurityBundle.builder().disableBufferLimitValidation();
+      return this;
+    }
+
+    @Override
+    public PlatformBundleBuilder<C> withCorsAllowedMethods(String... httpMethods) {
+      validateConfigureCors();
+      this.corsBundleBuilder.withAllowedMethods(httpMethods);
+      return this;
+    }
+
+    @Override
+    public PlatformBundleBuilder<C> withCorsAdditionalAllowedHeaders(
+        String... additionalAllowedHeaders) {
+      validateConfigureCors();
+      this.corsBundleBuilder.withAdditionalAllowedHeaders(additionalAllowedHeaders);
+      return this;
+    }
+
+    @Override
+    public PlatformBundleBuilder<C> withCorsAdditionalExposedHeaders(
+        String... additionalExposedHeaders) {
+      validateConfigureCors();
+      this.corsBundleBuilder.withAdditionalExposedHeaders(additionalExposedHeaders);
+      return this;
+    }
+
+    @Override
+    public PlatformBundleBuilder<C> withoutHalSupport() {
+      this.jacksonBundleBuilder.withoutHalSupport();
+      return this;
+    }
+
+    @Override
+    public PlatformBundleBuilder<C> withoutFieldFilter() {
+      this.jacksonBundleBuilder.withoutFieldFilter();
+      return this;
+    }
+
+    @Override
+    public PlatformBundleBuilder<C> withObjectMapperCustomization(
+        Consumer<ObjectMapper> customizer) {
+      this.jacksonBundleBuilder.withCustomization(customizer);
+      return this;
+    }
+
+    @Override
+    public PlatformBundleBuilder<C> alwaysWriteZonedDateTimeWithMillisInJson() {
+      this.jacksonBundleBuilder.alwaysWriteZonedDateTimeWithMillis();
+      return this;
+    }
+
+    @Override
+    public PlatformBundleBuilder<C> alwaysWriteZonedDateTimeWithoutMillisInJson() {
+      this.jacksonBundleBuilder.alwaysWriteZonedDateTimeWithoutMillis();
+      return this;
+    }
+
+    @Override
+    public OpenApiFinalBuilder<C> withExistingOpenAPI(String openApiJsonOrYaml) {
+      this.openApiBundleBuilder = getOpenApiBuilder().withExistingOpenAPI(openApiJsonOrYaml);
+      return this;
+    }
+
+    @Override
+    public OpenApiFinalBuilder<C> withExistingOpenAPIFromClasspathResource(String path) {
+      this.openApiBundleBuilder =
+          getOpenApiBuilder().withExistingOpenAPIFromClasspathResource(path);
+      return this;
+    }
+
+    @Override
+    public OpenApiFinalBuilder<C> addOpenApiResourcePackage(String resourcePackage) {
+      this.openApiBundleBuilder = getOpenApiBuilder().addResourcePackage(resourcePackage);
+      return this;
+    }
+
+    @Override
+    public OpenApiFinalBuilder<C> addOpenApiResourcePackageClass(Class<?> swaggerResourcePackage) {
+      this.openApiBundleBuilder =
+          getOpenApiBuilder().addResourcePackageClass(swaggerResourcePackage);
+      return this;
+    }
+
+    @Override
+    public OpenApiFinalBuilder<C> disableSwaggerEmbedParameter() {
```

Comment:
I never seen someone using it and nobody complained about it in the last 19 months it is present, so we should remove the option.

Since the OpenAPI module is still "experimental" we might remove it from there too.

Reply:
As the embed parameter is only documented if the response type has embeddable objects, this setting could lead to wrong documentation. In my opinion, it could be removed in both modules.

Reply:
If nobody is using this option, we should definitely remove it... Done :)

Reply:
>  In my opinion, it could be removed in both modules.

@JoergSiebahn isn't it a breaking change if we remove the method from the `sda-commons-server-starter` modul, even though nobody is actually using it?

Reply:
I think, we can safely remove it `disableEmbedParameter` from `OpenApiBundle` which is experimental and the new `SdaPlatformBundle` in `sda-commons-starter`.
You are right, we should NOT remove it in `sda-commons-server-starter` or `sda-commons-server-swagger`.
Maybe I should have clarified  _„both modules“_.

Reply:
Ok perfect, I've removed it from the `OpenApiBundle` & the `sda-commons-starter` builder..